cpu/ppc64_cpu_test.py

```diff
@@ -147,6 +152,14 @@ def dscr(self):
         command2 = "cat /sys/devices/system/cpu/dscr_default"
         self.equality_check("DSCR", command1, command2)
 
+    def smt_loop(self):
+        """
+        Tests smt on/off in a loop
+        """
+        for i in range(1, 100):
+            process.system_output("ppc64_cpu --smt=off && ppc64_cpu --smt=on",
```

Comment:
On a LPAR I have seen the SMT off operation takes roughly a minute .. while on a bare metal box it might take up 2 1/2 minutes. I am sure this value will vary depending on machine and resources.  I would like to avoid adding such a code.